src/reducers/errorReducer.js

```diff
@@ -0,0 +1,29 @@
+import produce from 'immer';
+
+const initialState = {};
+
+const errorReducer = produce((draft, action) => {
+  const { type, payload } = action;
+  const matches = /(.*)_(SUCCESS|ERROR)/.exec(type);
+
+  // not a *_SUCCESS / *_ERROR actions, so we ignore them
+  if (!matches) return;
+
+  const [, requestName, requestState] = matches;
+
+  switch (requestState) {
+    case 'ERROR':
+      draft[requestName] = payload?.message ? payload.message : '';
```

Comment:
[There's actually an easier way of doing this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Combining_with_the_nullish_coalescing_operator)

So it would be something like 
```js
draft[requestName] = payload?.message ?? '';
```

Reply:
Wow, cool, thanks Adrian! 👍 didn't know that one